applications/tari_dan_app_grpc/proto/transaction.proto

```diff
@@ -46,6 +46,7 @@ message Instruction {
     PUT_OUTPUT_IN_WORKSPACE = 2;
     EMIT_LOG = 3;
     CLAIM_BURN = 4;
+    EMIT_EVENT = 5;
```

Comment:
I am not convinced that we need a new Instruction type for events, as I understand events should be emitted only by components and not by the user. That would also mean that we don't need the extra fields in transaction nor the template_abi op

Reply:
Removed it